.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
I'd recommend naming this file differently to indicate that this is a Github action for pull requests